apps/labs/pages/api/annual-report.pdf.ts

```diff
@@ -0,0 +1,37 @@
+import type { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default async function handler(req: NextRequest) {
+  // TODO - remove after checking the logs on Vercel
+  console.log(`
+  req.url ${req.url}
+  req.referer ${req.headers.get('referer')}
+  req.ip ${req.ip}
+  user-agent ${req.headers.get('user-agent')}
+  `);
+  // The shape of this fetch comes from the Plausible docs:
+  // https://plausible.io/docs/events-api
+  await fetch('https://plausible.io/api/event', {
+    method: 'POST',
+    headers: {
+      'User-Agent': req.headers.get('user-agent'),
+      'X-Forwarded-For': req.ip,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      domain: 'labs.quansight.org',
+      name: 'pageview',
+      url: req.url,
+      referrer: req.headers.get('referer'),
```

Comment:
This is spelled `referer` everywhere in the file except here.

Are we API-constrained to use `referrer` as the key here, or could we harmonize to `referer`?

If we can't harmonize, I'd like a comment here to call out the intentional difference.